fuzzers/007-timing/bel/fixup_timings_txt.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import argparse
+
+
+def fix_line(line, site, filetype):
+    """
+    Squashes the entries for multilpe sites into one.
+    This is required when entries are defined for a different
+    site than they are reported.
+    Such situation happend e.g.
+    for BRAM_[LR]. All the entries are defined for RAMBFIFO36E1,
+    while they are reported for RAMB18E1 or FIFO18E1
+
+    Args:
+    line: raw dump file line
+    site: site to which all the entries will be squashed
+    filetype: entries type. One of [timings, pins, properties]
+
+    Returns:
+    newline: line with squashed entries
+    """
+
+    assert filetype in [
+        "timings", "pins", "properties"
+    ], "Unsupported filetype"
+
+    line = line.split()
+    newline = list()
+    sites_count = int(line[1])
+    newline.append(line[0])
+    # we'll emit only one site
+    newline.append("1")
```

Comment:
I think a short description of the input file format is required to make sense of this code.  Maybe add a description in a README?  Or something?  I find this code hard to review because it assumes a file structure.

Does it make any sense to first make a more structured form, and then manipulate the structured form?

Reply:
I added README to the BEL fuzzer describing how it wokrs and files format